### PR TITLE
Add CVE-2020-8935 through to CVE-2020-8944 for Asylo

### PR DESCRIPTION
**Pull request by pauldevt**

### CODE REVIEW

File: 2020/8xxx/CVE-2020-8935.json [hunk 1]
```diff
@@ -1,18 +1,108 @@
 {
```
Comment (cve-team) [inline 2020/8xxx/CVE-2020-8935.json:1]: 'version_value' is a required property

File: 2020/8xxx/CVE-2020-8938.json [hunk 2]
```diff
@@ -1,18 +1,110 @@
 {
```
Comment (cve-team) [inline 2020/8xxx/CVE-2020-8938.json:1]: 'version_value' is a required property

File: 2020/8xxx/CVE-2020-8939.json [hunk 3]
```diff
@@ -1,18 +1,110 @@
 {
```
Comment (cve-team) [inline 2020/8xxx/CVE-2020-8939.json:1]: 'version_value' is a required property

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

File: 2020/8xxx/CVE-2020-8935.json (hunk 1 above)
Comment (pauldevt) [reply 2020/8xxx/CVE-2020-8935.json:1]: Done

File: 2020/8xxx/CVE-2020-8938.json (hunk 2 above)
Comment (pauldevt) [reply 2020/8xxx/CVE-2020-8938.json:1]: Done

File: 2020/8xxx/CVE-2020-8939.json (hunk 3 above)
Comment (pauldevt) [reply 2020/8xxx/CVE-2020-8939.json:1]: Done

**Pull request merged**